scrapers/scrape_be_tests.py

```diff
@@ -4,13 +4,21 @@
 import re
 import scrape_common as sc
 
-html_url = 'https://www.besondere-lage.sites.be.ch/besondere-lage_sites/de/index/corona/index.html'
+html_url = 'https://www.besondere-lage.sites.be.ch/de/start/news/fallzahlen.html'
 d = sc.download(html_url, silent=True)
 
 soup = BeautifulSoup(d, 'html.parser')
 
+for caption in soup.find_all('caption'):
+    if caption.get_text() == 'Anzahl durchgeführte SARS-Cov-2 PCR-Tests':
+        weeklytable = caption.find_parents('table')
+    if caption.get_text() == 'Corona-Erkrankungen im Kanton Bern':
+        dailytable = caption.find_parents('table')
```

Comment:
Same here, something `soup.find('caption', string=re.compile('Anzahl durchgeführte SARS-Cov-2 PCR-Tests')).find_parent('table')` (or if you prefer `find_parents`, that would work as well).